src/scripts.js

```diff
@@ -1,13 +1,121 @@
-import './styles.css';
-import apiCalls from './apiCalls';
+import "./styles.css";
+import apiCalls from "./apiCalls";
+import RecipeRepository from "../src/classes/RecipeRepository";
+import recipes from "../src/data/recipes";
+import ingredients from "../src/data/ingredients";
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
-import "./images/What'sCookinLogo-01.png"
-import "./images/What'sCookinLogo-02.png"
-import "./images/chicken-leg.png"
-import "./images/star-icon-red.png"
-import "./images/star-icon-white.png"
-import "./images/star-icon-grey.png"
+import "./images/turing-logo.png";
+import "./images/What'sCookinLogo-01.png";
+import "./images/What'sCookinLogo-02.png";
+import "./images/chicken-leg.png";
+import "./images/star-icon-red.png";
+import "./images/star-icon-white.png";
+import "./images/star-icon-grey.png";
 
+console.log("Hello world");
 
-console.log('Hello world');
+//GlobalVariables
+let ingredData = ingredients;
+let recipeData = recipes;
+var recipeRepo = new RecipeRepository(ingredData, recipeData);
+
+//QuerySelectors
+const recipeList = document.querySelector(".recipe-list");
+const searchBox = document.querySelector(".search-box");
+const searchButton = document.querySelector(".search-button");
+const favoriteFilterBtn = document.querySelector(".filter-favorites-btn");
+const favoriteBtn = document.querySelector(".favorites-star");
+const recipeName = document.querySelector(".recipe-name");
+const dishImg = document.querySelector(".selected-dish-img");
+const directions = document.querySelector(".step-number");
+const recipeCost = document.querySelector(".recipe-cost");
+const listOfIngredients = document.querySelector(".list-of-ingredients");
+
+//EventListeners
+// favoriteFilterBtn.addEventListener('click');
+// favoriteBtn.addEventListener('click');
+searchBox.addEventListener("keypress", searchRecipe);
+
+recipeList.addEventListener("click", function (event) {
+  recipeRepo.repo.forEach((recipe) => {
+    if (event.target.parentNode.id === `${recipe.id}`) {
+      displayRecipe(recipe);
+    }
+  });
+});
+
+//Functions
+function searchRecipe() {
+  const tagSearched = recipeRepo.filterRecipeTag(searchBox.value);
```

Comment:
Future note: We need to fix this so the search functionality works. Maybe iterate through the input to determine if it is a tag or a name, then invoke the function. And maybe dont use the "Keypress". We'll need to experiment. 

An issue has been opened and we will solve later down the road.